tests/onegov/feriennet/test_views.py

```diff
@@ -1347,11 +1347,26 @@ def test_create_duplicate_notification(client):
 
 
 def test_import_account_statement(client, scenario):
+    scenario.add_user(username='member@example.org', role='member')
     scenario.add_period(confirmed=True)
     scenario.add_activity(title="Foobar", state='accepted')
-    scenario.add_occasion(cost=100)
+    scenario.add_occasion(cost=150)
+    scenario.add_attendee(name="Julian")
```

Comment:
You're creating ONE invoice with several invoice items (per child)... shouldn't we test with different invoices?